### PR TITLE
Fix classes and structures

### PR DESCRIPTION
**Pull request by Natchi59**

### CODE REVIEW

File: docs/doc/classes/EventsHandler.md [hunk 1]
```diff
@@ -4,40 +4,40 @@ Create an handler for bot events. The events can be in as many subfolders as des
...
 | --------- | ------------------------------------------------------------------------------------------------------- | ---------------------------------- | ------- | -------- |
-| client    | [ShewenyClient](./ShewenyClient.md) or [Client](https://discord.js.org/#/docs/main/stable/class/Client) | The client                         | None    |          |
 | directory | [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)       | The directory of events            | None    |          |
+| client    | [ShewenyClient](./ShewenyClient.md) or [Client](https://discord.js.org/#/docs/main/stable/class/Client) | The client                         | None    |          |
 | loadAll   | [Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)     | If handler load all in constructor | None    | ✓        |
 
 ## Properties
 
-### EventsHandler.client
+### E.client
```
Comment (Smaug6739) [inline docs/doc/classes/EventsHandler.md:14]: Remove the `E`

File: docs/doc/classes/ShewenyClient.md [hunk 2]
```diff
@@ -19,33 +19,33 @@ Return : [ClientHandlers](../typedef/ClientHandlers.md).
 
 ### commands
 
-Commands manager
+A collection of events or undefined if events are not loadeds.
```
Comment (Smaug6739) [inline docs/doc/classes/ShewenyClient.md:22]: Replace `events` by `commands`

File: docs/doc/classes/ShewenyClient.md [hunk 3]
```diff
@@ -59,31 +59,31 @@ Parameters :
 
 ### interactionCommmandCreate
 
-Emitted when an slash-command is created.
+Emitted when an Slash Command is created.
```
Comment (Smaug6739) [inline docs/doc/classes/ShewenyClient.md:62]: Slash-command or Context Menu

**Review by Smaug6739 — CHANGES_REQUESTED** [review]
Very small changes to make and it's good.

**Comment by Natchi59** [comment]
Done

**Review by Smaug6739 — APPROVED** [review]

**Pull request merged**